### ISSUE TITLE
Brush oxidation does not trigger with mending

### ISSUE THREAD

**Issue by solonovamax** [capsey/archeology-mod#68]
Because the [brush oxidation code](https://github.com/capsey/archeology-mod/blob/main/src/main/java/net/capsey/archeology/items/CopperBrushItem.java#L57) requires the brush reaching half-durability before oxidizing, if the brush has mending on it, it will never oxidize.

Instead, I recommend just storing the number of times it was used in the nbt data rather than relying on the durability.

**Comment by capsey** [comment]
But mending doesn't completely prevent decreasing durability, it just adds a way to replenish it. Or do you mean that experience got from successful brushing always gives more repair than lost brushing?

**Comment by capsey** [comment]
I think I made it that way intentionally as a way to allow players to keep brush on some oxidization level, but looking back, waxing would have been better solution.

**Comment by solonovamax** [comment]
> But mending doesn't completely prevent decreasing durability, it just adds a way to replenish it. Or do you mean that experience got from successful brushing always gives more repair than lost brushing?

yeah, it always repairs the brush to full, preventing the durability from ever decreasing.

> I think I made it that way intentionally as a way to allow players to keep brush on some oxidization level, but looking back, waxing would have been better solution.

I don't really see why anyone would want to keep the brush at a specific oxidation level, tbh, but idk


also, if the brush has unbreaking, that makes it take significantly longer to level up the brush as well. it really should just be a counter stored in nbt data.
